### PR TITLE
Refine document for new user to submit job

### PR DESCRIPTION
**Pull request by squirrelsc**
1. New guidance to submit job.
2. Refine homepage to connect with new guidance.
3. Reorganize content of troubleshooting for next refactoring.
4. Fix some links.

### CODE REVIEW

File: docs/user/training.md [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+    - [Job workflow](#job-workflow)
+  - [Reference](#reference)
+
+This document is for beginners, who use OpenPAI to train machine learning models or execute other commands.
+
+It assumes that you know IP address or domain name and have an account of OpenPAI. If there isn't an OpenPAI cluster yet, refer to [here](../../README.md#deploy-openpai) to deploy one.
+
+## Submit a hello-world job
+
+The **job** of OpenPAI defines how to execute command(s) in specified environment(s). A job can be model training, or other kinds of commands, even it's distributed on multiple servers.
```
Comment (scarlett2018) [inline docs/user/training.md]: even can be

File: README.md [hunk 2]
```diff
@@ -109,41 +109,23 @@ If FAQ doesn't resolve it, refer to [here](#get-involved) to ask question or sub
 
 ## Train models
 
-Like all machine learning platforms, OpenPAI is a production tool. To maximize utilization, it's recommended to submit training jobs and OpenPAI will allocate resource to run it. If there are too many jobs, some jobs may wait in the queue. This is different with training models on dedicated servers for each person, and it needs a bit more knowledge about how to submit/manage training jobs on OpenPAI.
+Like all machine learning platforms, OpenPAI is a production tool. To maximize utilization, it's recommended to submit training jobs and let OpenPAI to allocate resource and run it. If there are too many jobs, some jobs may wait in the queue, and OpenPAI choose some server(s) to run a job. This is different with run code on specified servers, and it needs a bit more knowledge about how to submit/manage training jobs on OpenPAI.
```
Comment (scarlett2018) [inline README.md]: a productivity tool
and run -> for running
queue, 后的一堆 -> queue till needed resources are allocated.
specified->dedicated

File: docs/user/training.md (hunk 1 above)
Comment (squirrelsc) [reply docs/user/training.md]: fixed

File: README.md (hunk 2 above)
Comment (squirrelsc) [reply README.md]: fixed part of them

**Pull request merged**